src/client/clientmap.cpp

```diff
@@ -400,10 +401,15 @@ void ClientMap::updateDrawList()
 					(look.Z > 0 ? 16 : 32);
 			
 			// This bitset is +Z-Z+Y-Y+X-X (See MapBlockMesh), and axis is XYZ.
-			u8 transparent_sides = (occlusion_culling_enabled && block) ? ~block->solid_sides : 0x3F;
+			// Get he block's transparent sides
+			u8 transparent_sides = (occlusion_culling_enabled && block && block_inner_sides != 0x3F) ? ~block->solid_sides : 0x3F;
```

Comment:
Now the camera block can't have opaque far sides.

Reply:
Indeed, good catch. Fixed now, thank you!